postgres.md

```diff
@@ -208,34 +208,38 @@ CREATE TABLE foo(bar integer, NOT NULL (bar));
 CREATE TABLE foo(bar integer NOT NULL);
 ```
 
-##### `PRIMARY KEY` constraints must be declared into the table declaration WITH explicit column reference.
+##### Contraints must de declared outside of its referencing table declaration. This allows a much more malleable code.

```

Comment:
To me it's more machine-friendly but less dev-friendly. That's basically a technical constraint.

Anyway, in “Contraints must de declared outside of its referencing table”, I'm not sure about the "referencing".